DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetAccountLoginPrompt.cs

```diff
@@ -0,0 +1,97 @@
+﻿using ClientGUI;
+using DTAClient.Domain.Multiplayer.CnCNet;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Input;
+using Rampastring.XNAUI;
+using Rampastring.XNAUI.Input;
+using Rampastring.XNAUI.XNAControls;
+using System;
+using System.Net;
+
+namespace DTAClient.DXGUI.Multiplayer.CnCNet
+{
+    public class CnCNetAccountLoginPrompt : XNAWindow
```

Comment:
With this many login windows, I'd throw a short `/// <summary>` comment here before each class to quickly clarify the purpose of the window.